load.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace HM\Platform\SSO;
+
+use function HM\Platform\get_config;
+use function HM\Platform\register_module;
+
+require_once __DIR__ . '/inc/saml/namespace.php';
```

Comment:
curious about this approach - any reason to load these by default and not just based on whether they're enabled?

Reply:
I think in discussions with @rmccue etc, we decided it was simpler to include all code ahead of time, regardless if used. The exception to that is side-effects inclusions, like wordpress plugin files

Reply:
Yeah, I think if there's no side effects, we should always include the code unconditionally. It slightly improves the opcache performance (although a micro-optimisation), and simplifies the code flow.

Reply:
Ok, I need to update some things in the SEO module to reflect that then 👍